src/sentry_value.h

```diff
@@ -1,6 +1,7 @@
 #ifndef SENTRY_VALUE_H_INCLUDED
 #define SENTRY_VALUE_H_INCLUDED
 
+#include "sentry.h"
```

Comment:
same here I guess… Not sure why you added this in the first place?

Reply:
i didn't, clangd did